source/lac/trilinos_sparse_matrix.cc

```diff
@@ -2310,6 +2310,17 @@ namespace TrilinosWrappers
     return ((sizeof(TrilinosScalar)+sizeof(TrilinosWrappers::types::int_type))*
             matrix->NumMyNonzeros() + sizeof(int)*local_size() + static_memory);
   }
+
+  const MPI_Comm &SparseMatrix::get_mpi_communicator () const
+  {
+    static MPI_Comm communicator;
```

Comment:
Yup, I agree. This was only to be consistent with what happens in PETSc matrices.